### ISSUE TITLE
Call latex-path in order to obtain LaTeX output

### ISSUE THREAD

**Issue by mardukbp** [soegaard/racket-cas#7]
While trying the examples in the README, I discovered that it is necessary to call

 `(latex-path "/path/to/pdflatex")` 

in order for `render` to work. It would be nice to add this so that people can experiment with this nice little CAS.

**Comment by soegaard** [comment]
Which OS do you use?
Which path do you have to pdflatex?

**Comment by mardukbp** [comment]
OS: Ubuntu 16.04

`/usr/bin/pdflatex`

**Comment by soegaard** [comment]
Hmm. I see that racket-poppler uses "latex" and not "pdflatex".

https://github.com/soegaard/racket-poppler/blob/master/racket-poppler/render-tex.rkt#L43

Maybe it is racket-poppler that needs to be changed?

**Comment by mardukbp** [comment]
Yes. Actually I learned about `latex-path` in the README of racket-poppler.
